internal/install/execution/terminal_reporter.go

```diff
@@ -87,25 +89,70 @@ func (r TerminalStatusReporter) InstallComplete(status *InstallStatus) error {
 		fmt.Println("  ---")
 	}
 
-	if status.hasAnyRecipeStatus(RecipeStatusTypes.INSTALLED) {
-		fmt.Println("  New Relic installation complete!")
+	hasInstalledRecipes := status.hasAnyRecipeStatus(RecipeStatusTypes.INSTALLED)
+	if hasInstalledRecipes {
+		fmt.Println("  New Relic installation complete")
+
+		printInstallationSummary(status)
 	}
 
 	linkToData := ""
 	if status.PlatformLinkGenerator != nil {
 		linkToData = status.PlatformLinkGenerator.GenerateRedirectURL(*status)
 	}
 
-	if linkToData != "" {
-		fmt.Printf("  Your data is available at %s", linkToData)
+	if hasInstalledRecipes && linkToData != "" {
+		fmt.Println("  Use the link below to view your data.")
+		fmt.Printf("  %s  %s", color.GreenString("\u2B95"), linkToData)
 	}
 
 	fmt.Println()
 
 	return nil
 }
 
+func printInstallationSummary(status *InstallStatus) {
+	fmt.Println()
+	fmt.Println("  --------------------")
+	fmt.Println("  Installation Summary")
+	fmt.Println()
+
+	successIcon := color.GreenString("\u2705")
+	failedIcon := color.RedString("\u274C")
+	neurtalIcon := color.RedString("\u2796")
+
+	for _, s := range status.Statuses {
+		var icon string
+		suffix := fmt.Sprintf("(%s)", strings.ToLower(string(s.Status)))
+
+		if s.Status == RecipeStatusTypes.INSTALLED {
+			icon = successIcon
+			suffix = fmt.Sprintf("(%s)", color.GreenString(strings.ToLower(string(s.Status))))
+		}
+
+		if s.Status == RecipeStatusTypes.FAILED {
+			icon = failedIcon
+			suffix = fmt.Sprintf("(%s)", color.RedString(strings.ToLower(string(s.Status))))
```

Comment:
@sanderblue we should check the color coding, Red might be viewed as too harsh (making users think of an error), maybe yellow, worth checking with Sam/Aaron and/or Daniel

Reply:
Updated to yellow and a less harsh icon :)